pkg/services/postgresqldb/types_dbms_only.go

```diff
@@ -0,0 +1,47 @@
+package postgresqldb
+
+import "github.com/Azure/open-service-broker-azure/pkg/service"
+
+// ServerProvisioningParameters encapsulates PostgreSQL-specific dbms
+// provisioning options
+type ServerProvisioningParameters struct {
```

Comment:
Can we change this to `DBMSOnlyProvisioningParameters`?

Reply:
Yeah good call. I originally had Server and Database embedded for the AllInOne and thought this read better, forgot to change the name.

Reply:
I kept this as ServerProvisioningParameters, I think it reads better in the embedding.